chat_shell/chat_shell/history/loader.py

```
@@ -637,20 +637,161 @@ def _build_document_text_prefix(
+        if not kb_kind:
+            logger.warning(
+                f"[_fetch_kb_documents_content] KB kind not found: id={knowledge_base_id}"
+            )
+            return ""
+
+        # Query all documents in this knowledge base
+        documents = (
+            db.query(KnowledgeDocument)
+            .filter(
+                KnowledgeDocument.kind_id == knowledge_base_id,
+                KnowledgeDocument.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `KnowledgeDocument.is_active:` for truth checks

Replace with `KnowledgeDocument.is_active`

(E712)